hud-data/spacehud.json

```diff
@@ -45,9 +46,14 @@
       "16/10"
     ]
   },
+  "tags": [
+     "minimalist", 
+     "simple", 
+   ]
   "repo": "https://github.com/BingBongBonky/SpaceHud",
   "hash": "9c5f6ac772d24ba261a5b3ff3bf72c59049408da",
   "resources": [
-    "spacehud-banner"
+    "spacehud-01-mainpage"
```

Comment:
I believe you need a comma after `"spacehud-01-mainpage"` since you're adding a new resource below it